lib/desktop_screens_new/trusted_senders_screen/widgets/desktop_contact_tile.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:atsign_atmosphere_pro/utils/text_styles.dart';
+import 'package:atsign_atmosphere_pro/view_models/trusted_sender_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class DesktopContactTile extends StatelessWidget {
+  const DesktopContactTile({
+    Key? key,
+    required this.title,
+    required this.subTitle,
+    this.showImage = false,
+    this.image,
+  }) : super(key: key);
+
+  final String? title, subTitle;
+  final bool showImage;
+  final Uint8List? image;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        color: Colors.white,
+        borderRadius: BorderRadius.circular(10),
+      ),
+      clipBehavior: Clip.hardEdge,
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          showImage
+              ? Container(
+                  height: 60,
+                  width: 60,
+                  child: Image.memory(
+                    image!,
+                    fit: BoxFit.fill,
+                    errorBuilder: (BuildContext _context, _, __) {
+                      return Container(
+                        child: Icon(
+                          Icons.image,
+                          size: 30.toFont,
+                        ),
+                      );
+                    },
+                  ),
+                )
+              : ContactInitial(
+                  initials: title ?? ' ',
+                  size: 60,
+                  maxSize: (80.0 - 20.0),
+                  minSize: 60,
+                  borderRadius: 0,
+                ),
+          SizedBox(width: 20),
+          Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SizedBox(width: 10.toHeight),
+              Text(
+                title!,
+                style: TextStyle(
+                  color: Colors.black,
+                  fontWeight: FontWeight.bold,
+                  fontSize: 16,
+                ),
+                maxLines: 1,
+                overflow: TextOverflow.ellipsis,
+              ),
+              subTitle != null
+                  ? Text(
+                      subTitle!,
+                      style: CustomTextStyles.desktopPrimaryRegular12,
+                      maxLines: 1,
+                      overflow: TextOverflow.ellipsis,
+                    )
+                  : SizedBox(),
+              SizedBox(width: 10.toHeight),
+            ],
+          ),
+          Spacer(),
+          Icon(
+            Icons.send_outlined,
+            color: Theme.of(context).primaryColor,
+            size: 24,
+          ),
+          const SizedBox(
+            width: 10,
+          ),
+          InkWell(
+            onTap: () async {
```

Comment:
Can we separate the onTap function outside UI ?

Reply:
fixed